pkg/container/kubernetes/client.go

```diff
@@ -66,9 +81,77 @@ func NewClient(_ context.Context) (*Client, error) {
 	return &Client{
 		runtimeType: runtime.TypeKubernetes,
 		client:      clientset,
+		config:      config,
 	}, nil
 }
 
+// createRemoteClusterConfig creates a Kubernetes config for remote cluster access
+func createRemoteClusterConfig() (*rest.Config, error) {
+	// Get the kubeconfig path from environment variable or use default
+	kubeconfigPath := os.Getenv("KUBECONFIG")
+	if kubeconfigPath == "" {
+		// Use default kubeconfig path
+		homeDir, err := os.UserHomeDir()
+		if err != nil {
+			return nil, fmt.Errorf("failed to get user home directory: %w", err)
+		}
+		kubeconfigPath = homeDir + "/.kube/config"
```

Comment:
hmm, this will not work on Windows will it? I was browsing through the clientcmd package because I was not familiar with it and noticed that it exports RecommendedHomeFile, could we use that?